docs/source/models/gbd2017/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -47,27 +47,70 @@ represents **all** forms of anemia that are attributable to several causes,
 including causes such as hemoglobinopathies and hemolytic anemias that are not 
 causally attributable to iron deficiency. 
 
-Iron Deficiency Risk Factor
-+++++++++++++++++++++++++++
-
-The iron deficiency risk factor in GBD 2017 is modeled as a distribution of 
-hemoglobin concentrations. The risk factor is a **continuous measure**.
-
-.. todo::
-
-	Describe the population and distribution type (normal vs. ensemble)
+Anemia Impairment
++++++++++++++++++
+
+The anemia impairment in GBD 2017 represents the total prevalence of anemia due 
+to all causes modeled in GBD (ex: dietary iron deficiency anemia, anemia due to 
+maternal hemorrhage, sickle cell anemia, etc.). Estimating the total prevalence 
+of the anemia impairment for a given population is the first step in modeling 
+anemia in GBD 2017. This is done by fitting a distribution of hemoglobin levels 
+for that population from primary input data. For GBD 2017, an ensemble 
+distribution was used, which was 40% gamma and 60% mirror gumbel. Source code 
+for this process is available `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia/browse/model/envelope>`_.
+
+Once a the distribution is fit to hemoglobin levels for a particular age-, 
+sex-, and location-specific demographic group, age- and sex-specific hemoglobin 
+thresholds, as defined by the WHO and described in the table below, are used to 
+determine the prevalence of anemia by severity for that population.
 
-Modeling Strategy for the Iron Deficiency Risk Factor
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. list-table:: Hemoglobin Thresholds (g/L)
```

Comment:
Perhaps specify in the table title that these are WHO thresholds, and cite the source (either WHO or the YLD appendix). I googled "who hemoglobin thresholds", and this came up:

https://www.who.int/vmnis/indicators/haemoglobin.pdf

The thresholds and age groups it lists are slightly different, but it looks like it's from 2011, so perhaps there is a more recent version?

Reply:
Updated in latest commit